examples/vercel-ai-chatbot/app/api/chat/route.ts

```diff
@@ -78,8 +78,13 @@ export async function POST(req: Request) {
   const stream = new ReadableStream({
     async pull(controller) {
       const encoder = new TextEncoder()
-      const emit = (event: unknown) =>
-        controller.enqueue(encoder.encode(JSON.stringify(event) + '\n'))
+      const emit = (event: unknown) => {
```

Comment:
[nit] Maybe worth a comment here stating a shortened version of what's stated in the video so someone using this as an example knows what's going